api/src/test/java/org/projectnessie/tools/catalog/migration/api/ITHiveToNessieCatalogMigrator.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.tools.catalog.migration.api;
+
+import java.util.Collections;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.projectnessie.tools.catalog.migration.api.test.HiveMetaStoreRunner;
+
+public class ITHiveToNessieCatalogMigrator extends AbstractTestCatalogMigrator {
+
+  @BeforeAll
+  protected static void setup() throws Exception {
+    HiveMetaStoreRunner.startMetastore();
+
+    sourceCatalog = HiveMetaStoreRunner.hiveCatalog();
```

Comment:
I later realized that, I can directly use `HiveMetaStoreRunner.hiveCatalog()` as the catalog instead of building one more copy. 

I will use `initializeSourceCatalog` and `initializeTargetCatalog` style itself as it matches other catalog testcase styles.